core/timeline_api.php

```diff
@@ -101,11 +112,16 @@ function timeline_events( $p_start_time, $p_end_time, $p_max_events, $p_filter =
 						break;
 				}
 				break;
+			case FILE_ADDED:
+ 				$t_bugnote_id = $t_history_event['old_value'];
+				$t_event = new IssueFileAddedTimelineEvent( $t_timestamp, $t_user_id, $t_issue_id, $t_bugnote_id );
```

Comment:
From class IssueFileAddedTimelineEvent
`public function __construct( $p_timestamp, $p_user_id, $p_issue_id ) `
`$t_bugnote_id` is not needed.